mne/gui/tests/test_kit2fiff_gui.py

```diff
@@ -71,23 +81,23 @@ def test_kit2fiff_model():
     model.stim_slope = '+'
     events_bin = mne.find_events(raw_bin, stim_channel='STI 014')
 
-    model.stim_chs = '<'
+    model.stim_coding = '<'
     raw = model.get_raw()
     events = mne.find_events(raw, stim_channel='STI 014')
     assert_array_equal(events, events_bin)
 
     events_rev = events_bin.copy()
     events_rev[:, 2] = 1
-    model.stim_chs = '>'
+    model.stim_coding = '>'
     raw = model.get_raw()
     events = mne.find_events(raw, stim_channel='STI 014')
     assert_array_equal(events, events_rev)
 
-    model.stim_chs = 'man'
-    model.stim_chs_manual = list(range(167, 159, -1))
+    model.stim_coding = 'channel'
+    model.stim_chs = "160:161"
```

Comment:
What part is confusing? It's testing the new, more flexible way of specifying stim channels with a string.